src/Directory.Build.props

```diff
@@ -12,13 +12,18 @@
 
   <PropertyGroup Condition="'$(Language)' == 'C#'">
     <CheckForOverflowUnderflow>false</CheckForOverflowUnderflow>
+    <!-- Build machines only have Visual Studio 15.5 -->
     <LangVersion>7.2</LangVersion>
+    <Features>strict</Features>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Language)' == 'VB'">
+    <!-- Build machines only have Visual Studio 15.5 (which comes with VB 15.3) -->
+    <LangVersion>15.3</LangVersion>
```

Comment:
We should be building with a compiler version from nuget not VS. Do 7.3 and 15.5 give errors?

Reply:
When I looked at it before, we are for the main build, but not for the xaml compiler build in markuppass1 :(

Reply:
Sounds like a bug? Is there one on this?

Reply:
Not sure?  @tmat?

Reply:
Could be that `Language` is not set properly in tmp_proj. .NET 4.7.2 changed the name of the temp project, so our detection of the temp proj in the toolset stopped working. I fixed that in recent version of the toolset.